web/index.html

```diff
@@ -97,14 +97,11 @@
     <!-- Pre-connect to backend to improve performance -->
     <link rel="preconnect" href="https://app-backend.electricitymaps.com/" />
     <link rel="preconnect" href="https://o192958.ingest.sentry.io" />
-    <link rel="preconnect" href="https://plausible.io" />
```

Comment:
We have a bunch of analytics that pull data from Plausible, both the commercial website and the app. We might want to leave this in place and turn it off after we've stopped paying (if that's the plan).

Best to speak with Jesper and Oli around how we retire the use of Plausible.